clients/package.json

```
@@ -2,7 +2,8 @@
   "workspaces": [
     "admin-ui",
     "fides-js",
-    "privacy-center"
+    "privacy-center",
+    "fidesui"
```

nit: When in doubt, I always prefer an alphabetic sort:
```suggestion
    "fidesui",
    "privacy-center"
```

(NOTE: This nitpick is really unnecessary but I'm leaving it hopefully so you feel _comforted_ knowing that I am looking at every file!)

---

good call. And keep the nitpicks coming. I appreciate them!